### PR TITLE
#34: Added choice of installation profile.

### PR DESCRIPTION
**Pull request by Decipher**
As per #34. Could probably use some tests, but I don't know how best to do that as I assume you can't have separate behat tests per job.

### CODE REVIEW

File: environments/drupal-7.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 function drupal_ti_install_drupal() {
 	drush --yes dl drupal-7 --drupal-project-rename=drupal
 	cd drupal
-	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install testing --db-url="$DRUPAL_TI_DB_URL"
+	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install $DRUPAL_TI_INSTALL_PROFILE --db-url="$DRUPAL_TI_DB_URL"
```
Comment (LionsAd) [inline environments/drupal-7.sh]: That is a nit and you can't know, but bash variables are enclosed by "" in this project.

Comment (Decipher) [inline environments/drupal-7.sh]: Will resolve ASAP.

Comment (LionsAd) [inline environments/drupal-7.sh]: Any update?

Comment (Decipher) [inline environments/drupal-7.sh]: Sorry, been focusing on getting File (Field) Paths a long overdue release,
I will get back to this soon, if not at DrupalCon.

--

http://stuar.tc/lark
@Decipher

**Pull request closed without merging**